components/public-api-server/main.go

```diff
@@ -6,15 +6,66 @@ package main
 
 import (
 	"fmt"
+	"net"
 	"net/http"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"google.golang.org/grpc"
 )
 
 func main() {
-	addr := ":9000"
-	err := http.ListenAndServe(addr, http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-		_, _ = w.Write([]byte(`hello world`))
-	}))
+	errors := make(chan error)
+	defer close(errors)
+
+	go func() {
+		addr := ":9000"
+		fmt.Println("Serving HTTP", addr)
+		err := http.ListenAndServe(addr, http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+			_, _ = w.Write([]byte(`hello world\n`))
+		}))
```

Comment:
Do we still need this http server in addition to the GRPC server below?

Reply:
Yes. We'll need to serve metrics, ready/health endpoints etc